mono_repo.yaml

```diff
@@ -6,4 +6,3 @@ github:
 
 merge_stages:
 - analyze_and_format
-
```

Comment:
https://yamllint.readthedocs.io/en/stable/rules.html#module-yamllint.rules.new_line_at_end_of_file

Reply:
There was already a newline. You added an extra one. I thought it was a mistake

Reply:
Ah yeah, my bad -- thanks!